oteapi/models/mappingconfig.py

```diff
@@ -21,7 +19,7 @@ class MappingConfig(GenericConfig):
             "given as local value/IRI-expansion-pairs."
         ),
     )
-    triples: Optional[List[SemanticTriple]] = Field(
+    triples: Optional[List[Tuple[str, str, str]]] = Field(
```

Comment:
I don't get that locally? Using `typing.Annotated` should exactly fix this issue? Hmm - interesting.

Reply:
It could be that I have an older version pydantic. For now I suggest that we go for the list of tuples. We can reintroduce the `SemanticTriple` when we have a requirement of pydantic that we for sure supports it.